toolchain/testing/file_test.cpp

```diff
@@ -88,7 +88,9 @@ class ToolchainFileTest : public FileTestBase {
  private:
   // Returns the toolchain subdirectory being tested.
   static auto GetComponent(llvm::StringRef test_name) -> llvm::StringRef {
-    CARBON_CHECK(test_name.consume_front("toolchain/"));
+    auto pos = test_name.find("toolchain/");
```

Comment:
```suggestion
    // This handles cases where the toolchain directory may be in a repository but not at the root.
    auto pos = test_name.find("toolchain/");
```

What do you think of commenting why this isn't the more trivial consume? That way it's less likely to get "simplified". :)